react/components/LogoutCustomerSession.tsx

```diff
@@ -100,15 +109,17 @@ export default class LogoutCustomerSession extends Component<Props> {
           {mobile ? this.clientName : client.email}
         </div>
       </div>
-    )
-  }
+    );
```

Comment:
Are you sure about the addition of semi colons back?

Reply:
Oh, the `.prettierrc` is missing in this component. i'll add that